deploy/ForeignControllerInit.sol

```diff
@@ -63,7 +70,15 @@ library ForeignControllerInit {
 
         // Step 2: Initialize the controller
 
-        _initController(controllerInst, configAddresses, checkAddresses, mintRecipients);
+        InitParams[] memory params = new InitParams[](1);
+        params[0] = InitParams({
```

Comment:
We don't need this struct

Reply:
SHould just be the one change in the `ConfigAddressParams` struct and the for loop

Reply:
fixed